cypher-shell/src/main/java/org/neo4j/shell/DatabaseManager.java

```diff
@@ -7,7 +7,7 @@
  */
 public interface DatabaseManager
 {
-    String ABSENT_DB_NAME = "";
+    String ABSENT_DB_NAME = "neo4j"; // TODO After beta02, "" and literal null are no longer valid database names.
```

Comment:
This will break the internal Cypher Shell logic of distinguishing between having the database name set to an explicit name and having it set to whatever is the default.

We have already adapted to never pass the empty string to the driver a long time ago (See `BoltStateHandler.reconnect()`).